ui/src/components/widgets/Console.vue

```diff
@@ -18,7 +18,7 @@
 <template>
   <a
     v-if="['vm', 'systemvm', 'router', 'ilbvm'].includes($route.meta.name) && 'updateVirtualMachine' in $store.getters.apis"
-    :href="'/client/console?cmd=access&vm=' + resource.id"
+    :href="server + '/client/console?cmd=access&vm=' + resource.id"
```

Comment:
@utchoang `/client` can be changed in config.json. Would be better to pick it up from apiBase and trim off the /api